extensions/functions_arithmetic.yaml

```diff
@@ -1614,3 +1614,131 @@ window_functions:
         decomposable: NONE
         return: i64?
         window_type: PARTITION
+  - name: "first_value"
+    description: >
+      The `expression` is evaluated against the first row.  The `expression` can be a
+      column, expression or subquery that evaluates to a single value.
+    impls:
+      - args:
+          - value: any1
+            name: expression
+        nullability: DECLARED_OUTPUT
+        decomposable: NONE
+        return: any1?
+        window_type: PARTITION
+  - name: "last_value"
+    description: >
+      The `expression` is evaluated against the last row.  The `expression` can be a
+      column, expression, or subquery that evaluates to a single value.
+    impls:
+      - args:
+          - value: any1
+            name: expression
+        nullability: DECLARED_OUTPUT
+        decomposable: NONE
+        return: any1?
+        window_type: PARTITION
+  - name: "nth_value"
+    description: >
+      The `expression` is evaluated against a row that comes after the current row based
+      on the `offset`.  `offset` should be a positive integer.  The `expression` can be a
+      column, expression or subquery that evaluates to a single value.
```

Comment:
This is incorrect.  Also, we shouldn't use the term `offset` as it is ambiguous.  Perhaps `window_offset`.  Given a window of `[A, B, C, D, E, F, G]` and the call `nth_value(col, 2)` then I would expect, based on this description, that the return value would depend on the current row.  In fact, The description here reads very much like `lead`.  In other words, I would expect the return would be `[C, D, E, F, G, NULL, NULL]` but that is incorrect.

It's more correct to say that it grabs "the nth row in the window".  The return in the above example should be `[B, B, B, B, B, B, B]`